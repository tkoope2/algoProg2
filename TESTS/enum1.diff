7a8,135
> @: a1 a2 a3 a4 a5 a6 a7 a8
> @: a1 a2 a3 a4 a5 a6 b7 a8
> @: a1 a2 a3 a4 a5 b6 a7 a8
> @: a1 a2 a3 a4 a5 b6 b7 a8
> @: a1 a2 a3 a4 b5 a6 a7 a8
> @: a1 a2 a3 a4 b5 a6 b7 a8
> @: a1 a2 a3 a4 b5 b6 a7 a8
> @: a1 a2 a3 a4 b5 b6 b7 a8
> @: a1 a2 a3 b4 a5 a6 a7 a8
> @: a1 a2 a3 b4 a5 a6 b7 a8
> @: a1 a2 a3 b4 a5 b6 a7 a8
> @: a1 a2 a3 b4 a5 b6 b7 a8
> @: a1 a2 a3 b4 b5 a6 a7 a8
> @: a1 a2 a3 b4 b5 a6 b7 a8
> @: a1 a2 a3 b4 b5 b6 a7 a8
> @: a1 a2 a3 b4 b5 b6 b7 a8
> @: a1 a2 b3 a4 a5 a6 a7 a8
> @: a1 a2 b3 a4 a5 a6 b7 a8
> @: a1 a2 b3 a4 a5 b6 a7 a8
> @: a1 a2 b3 a4 a5 b6 b7 a8
> @: a1 a2 b3 a4 b5 a6 a7 a8
> @: a1 a2 b3 a4 b5 a6 b7 a8
> @: a1 a2 b3 a4 b5 b6 a7 a8
> @: a1 a2 b3 a4 b5 b6 b7 a8
> @: a1 a2 b3 b4 a5 a6 a7 a8
> @: a1 a2 b3 b4 a5 a6 b7 a8
> @: a1 a2 b3 b4 a5 b6 a7 a8
> @: a1 a2 b3 b4 a5 b6 b7 a8
> @: a1 a2 b3 b4 b5 a6 a7 a8
> @: a1 a2 b3 b4 b5 a6 b7 a8
> @: a1 a2 b3 b4 b5 b6 a7 a8
> @: a1 a2 b3 b4 b5 b6 b7 a8
> @: a1 b2 a3 a4 a5 a6 a7 a8
> @: a1 b2 a3 a4 a5 a6 b7 a8
> @: a1 b2 a3 a4 a5 b6 a7 a8
> @: a1 b2 a3 a4 a5 b6 b7 a8
> @: a1 b2 a3 a4 b5 a6 a7 a8
> @: a1 b2 a3 a4 b5 a6 b7 a8
> @: a1 b2 a3 a4 b5 b6 a7 a8
> @: a1 b2 a3 a4 b5 b6 b7 a8
> @: a1 b2 a3 b4 a5 a6 a7 a8
> @: a1 b2 a3 b4 a5 a6 b7 a8
> @: a1 b2 a3 b4 a5 b6 a7 a8
> @: a1 b2 a3 b4 a5 b6 b7 a8
> @: a1 b2 a3 b4 b5 a6 a7 a8
> @: a1 b2 a3 b4 b5 a6 b7 a8
> @: a1 b2 a3 b4 b5 b6 a7 a8
> @: a1 b2 a3 b4 b5 b6 b7 a8
> @: a1 b2 b3 a4 a5 a6 a7 a8
> @: a1 b2 b3 a4 a5 a6 b7 a8
> @: a1 b2 b3 a4 a5 b6 a7 a8
> @: a1 b2 b3 a4 a5 b6 b7 a8
> @: a1 b2 b3 a4 b5 a6 a7 a8
> @: a1 b2 b3 a4 b5 a6 b7 a8
> @: a1 b2 b3 a4 b5 b6 a7 a8
> @: a1 b2 b3 a4 b5 b6 b7 a8
> @: a1 b2 b3 b4 a5 a6 a7 a8
> @: a1 b2 b3 b4 a5 a6 b7 a8
> @: a1 b2 b3 b4 a5 b6 a7 a8
> @: a1 b2 b3 b4 a5 b6 b7 a8
> @: a1 b2 b3 b4 b5 a6 a7 a8
> @: a1 b2 b3 b4 b5 a6 b7 a8
> @: a1 b2 b3 b4 b5 b6 a7 a8
> @: a1 b2 b3 b4 b5 b6 b7 a8
> @: b1 a2 a3 a4 a5 a6 a7 a8
> @: b1 a2 a3 a4 a5 a6 b7 a8
> @: b1 a2 a3 a4 a5 b6 a7 a8
> @: b1 a2 a3 a4 a5 b6 b7 a8
> @: b1 a2 a3 a4 b5 a6 a7 a8
> @: b1 a2 a3 a4 b5 a6 b7 a8
> @: b1 a2 a3 a4 b5 b6 a7 a8
> @: b1 a2 a3 a4 b5 b6 b7 a8
> @: b1 a2 a3 b4 a5 a6 a7 a8
> @: b1 a2 a3 b4 a5 a6 b7 a8
> @: b1 a2 a3 b4 a5 b6 a7 a8
> @: b1 a2 a3 b4 a5 b6 b7 a8
> @: b1 a2 a3 b4 b5 a6 a7 a8
> @: b1 a2 a3 b4 b5 a6 b7 a8
> @: b1 a2 a3 b4 b5 b6 a7 a8
> @: b1 a2 a3 b4 b5 b6 b7 a8
> @: b1 a2 b3 a4 a5 a6 a7 a8
> @: b1 a2 b3 a4 a5 a6 b7 a8
> @: b1 a2 b3 a4 a5 b6 a7 a8
> @: b1 a2 b3 a4 a5 b6 b7 a8
> @: b1 a2 b3 a4 b5 a6 a7 a8
> @: b1 a2 b3 a4 b5 a6 b7 a8
> @: b1 a2 b3 a4 b5 b6 a7 a8
> @: b1 a2 b3 a4 b5 b6 b7 a8
> @: b1 a2 b3 b4 a5 a6 a7 a8
> @: b1 a2 b3 b4 a5 a6 b7 a8
> @: b1 a2 b3 b4 a5 b6 a7 a8
> @: b1 a2 b3 b4 a5 b6 b7 a8
> @: b1 a2 b3 b4 b5 a6 a7 a8
> @: b1 a2 b3 b4 b5 a6 b7 a8
> @: b1 a2 b3 b4 b5 b6 a7 a8
> @: b1 a2 b3 b4 b5 b6 b7 a8
> @: b1 b2 a3 a4 a5 a6 a7 a8
> @: b1 b2 a3 a4 a5 a6 b7 a8
> @: b1 b2 a3 a4 a5 b6 a7 a8
> @: b1 b2 a3 a4 a5 b6 b7 a8
> @: b1 b2 a3 a4 b5 a6 a7 a8
> @: b1 b2 a3 a4 b5 a6 b7 a8
> @: b1 b2 a3 a4 b5 b6 a7 a8
> @: b1 b2 a3 a4 b5 b6 b7 a8
> @: b1 b2 a3 b4 a5 a6 a7 a8
> @: b1 b2 a3 b4 a5 a6 b7 a8
> @: b1 b2 a3 b4 a5 b6 a7 a8
> @: b1 b2 a3 b4 a5 b6 b7 a8
> @: b1 b2 a3 b4 b5 a6 a7 a8
> @: b1 b2 a3 b4 b5 a6 b7 a8
> @: b1 b2 a3 b4 b5 b6 a7 a8
> @: b1 b2 a3 b4 b5 b6 b7 a8
> @: b1 b2 b3 a4 a5 a6 a7 a8
> @: b1 b2 b3 a4 a5 a6 b7 a8
> @: b1 b2 b3 a4 a5 b6 a7 a8
> @: b1 b2 b3 a4 a5 b6 b7 a8
> @: b1 b2 b3 a4 b5 a6 a7 a8
> @: b1 b2 b3 a4 b5 a6 b7 a8
> @: b1 b2 b3 a4 b5 b6 a7 a8
> @: b1 b2 b3 a4 b5 b6 b7 a8
> @: b1 b2 b3 b4 a5 a6 a7 a8
> @: b1 b2 b3 b4 a5 a6 b7 a8
> @: b1 b2 b3 b4 a5 b6 a7 a8
> @: b1 b2 b3 b4 a5 b6 b7 a8
> @: b1 b2 b3 b4 b5 a6 a7 a8
> @: b1 b2 b3 b4 b5 a6 b7 a8
> @: b1 b2 b3 b4 b5 b6 a7 a8
> @: b1 b2 b3 b4 b5 b6 b7 a8
