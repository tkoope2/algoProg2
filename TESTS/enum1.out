@1++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
@2
@3 COMMAND-LINE:
@4
@5    ./epaths ladder10.g a8 
@6
@7++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
